narwhal/node/src/worker_node.rs

```diff
@@ -114,7 +119,11 @@ impl WorkerNodeInner {
 
         self.swap_registry(None);
 
-        info!("Narwhal worker {} shutdown is complete", self.id);
+        info!(
+            "Narwhal worker {} shutdown is complete - took {} seconds",
+            self.id,
+            now.elapsed().as_secs_f64()
```

Comment:
Could we add actual metric for narwhal start and shutdown time?

Reply:
Good point @lxfind , I've actually included a monitored scope https://github.com/MystenLabs/sui/pull/7316/files#diff-bf2537669f62dda9468ce278fb4a94eb48610029c5121ccd4a6cf44152fac60dR89 that will allow us to measure the execution time, unless you had in mind a dedicated metric ?

Reply:
Ah good to know. Assume they will show up in grafana?

Reply:
Absolutely, they'll be under the `monitored_scope_duration_ns` with the corresponding label name